open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -560,13 +562,18 @@ DEFINE
  CONFIGPROGRAMDATE         = DATE
  CONFIGPROGRAMMINDATE      = DATE
  CONFIGPROGRAMMISMATCH     = NUMBER
+
+ PARAMINSTALLDATE          = DATE
 END
 
 SETUP
- PROGRAMVERSION="1.3.1"
- LASTMODDATE='03/28/24'
- LASTMODTIME="15:00 MT"
- PROGRAMUPDATENOTE1="malformed json fix"
+ PROGRAMVERSION="1.4.0"
+ LASTMODDATE='05/14/24'
+ LASTMODTIME="12:00 MT"
+
+ PARAMINSTALLDATE='05/14/24'
```

Comment:
Is this correct?  Shouldn't the code just be pulling the PARAMINSTALLDATE from the CFG?  OH....maybe because we have the CFG program for this one?  And that is not being updated?

Reply:
I see this discussed in another thread...will respond there.